purchase.adoc

```diff
@@ -388,288 +517,533 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
         "<https://purchase.izettle.com/purchases/v2?limit=10&descending=true&lastPurchaseHash=1485858080407AOjsYOefEeax5pDdmnaGXw>; rel=\"next\""
   ]
 }
-```
+----
+--
+
+'''
+
+====
+
+==== Fetch purchase by id
```

Comment:
Fetch a purchase history

Reply:
Not sure what you mean here.
This endpoint is clearly `get one purchase by id`